code/modules/religion/burdened/psyker.dm

```diff
@@ -217,29 +245,36 @@
 	name = "Refill"
 	desc = "Perform a prayer, to refill your weapon."
 
-/obj/item/ammo_box/magazine/internal/cylinder/rev77
+/obj/item/ammo_box/magazine/internal/cylinder/revchap
 	name = "chaplain revolver cylinder"
-	ammo_type = /obj/item/ammo_casing/c77
-	caliber = CALIBER_77
+	ammo_type = /obj/item/ammo_casing/c38/holy
+	caliber = CALIBER_38
 	max_ammo = 5
 
-/obj/item/ammo_casing/c77
-	name = ".77 bullet casing"
-	desc = "A .77 bullet casing."
-	caliber = CALIBER_77
-	projectile_type = /obj/projectile/bullet/c77
+/obj/item/ammo_casing/c38/holy
+	name = "lucky .38 bullet casing"
+	desc = "A lucky .38 bullet casing. You feel lucky just holding it."
+	caliber = CALIBER_38
+	projectile_type = /obj/projectile/bullet/c38/holy
 	custom_materials = null
 
-/obj/projectile/bullet/c77
-	name = ".77 bullet"
-	damage = 18
+/obj/projectile/bullet/c38/holy
+	name = "lucky .38 bullet"
 	ricochets_max = 2
 	ricochet_chance = 50
 	ricochet_auto_aim_angle = 10
 	ricochet_auto_aim_range = 3
 	wound_bonus = -10
 	embedding = null
 
+/obj/projectile/bullet/c38/holy/on_hit(atom/target, blocked, pierce_hit)
+	. = ..()
+	var/roll_them_bones = pick(rand(1,38))
```

Comment:
```suggestion
	var/roll_them_bones = rand(1,38)
```

the pick is completely useless here

Reply:
I'm dense as a brick and I noticed it literally as I went back and looked through my code haha